runtime/autoload/dist/vimindent.vim

```diff
@@ -839,6 +849,25 @@ def CacheBracketBlock(line_A: dict<any>) # {{{2
     RegisterCacheInvalidation()
 enddef
 
+def CacheEnumBlock(line_A: dict<any>) # {{{2
+    if line_A.text !~ '^\s*enum\s'
+        return
+    endif
+
+    var pos: list<number> = getcurpos()
+    var startlnum: number = line_A.lnum + 1
+    var endlnum: number = search($'^\s*endenum$', 'nW') - 1
```

Comment:
Do you really need string interpolation here?

Reply:
Comments can follow `endenum`s, so `'^\C\s*endenum\>'`; else,  
you run the risk of matching some unrelated `endenum`.

Reply:
I am not very familiar with Vim’s regular expressions. Thank you for the guidance. I've corrected it.